apps/api/scripts/buildAkashTemplatesCache.ts

```diff
@@ -9,7 +9,7 @@ if (!githubPAT) {
   process.exit(1);
 }
 
-getTemplateGallery({ githubPAT, dataFolderPath: "./dist/.data" }).catch(err => {
+getTemplateGallery().catch(err => {
```

Comment:
the build is failing because it tries to prebuild templates and cannot because it requires full application configuration to be provided. But here we just need `GITHUB_PAT`. That's why this function accepted `githubPAT` parameter